main/FatesHistoryInterfaceMod.F90

```diff
@@ -1752,6 +1758,8 @@ subroutine update_history_dyn(this,nc,nsites,sites)
 
                   associate( scpf => ccohort%size_by_pft_class, &
                              scls => ccohort%size_class )
+			     
+		    gpp_cached = hio_gpp_si_scpf(io_si,scpf)
```

Comment:
@rgknox  I have had the busiest time for many years; besices relocating the whole family, there are four proposals to finish before the middle of March. For the issue here, this will give simulated d13C = 0 for the first time step. the values should be reasonable after several time steps as long as Ci is reasonably simulated.  So i guess this is not a big issue and putting a note should be sufficient at this point. Sorry for late response I will put most of my time on FATES after the mid of March. Thanks.